### PR TITLE
Add Ballerina MySQL Connector for the MySQL Connectors List

### PR DESCRIPTION
**Pull request by SachinAkash01**
## Description
[Ballerina](https://ballerina.io/) is a modern programming language designed to simplify the development of cloud-native applications by integrating seamlessly with network protocols and data formats. The Ballerina MySQL connector enhances this experience by providing developers with the ability to programmatically interact with MySQL.

Reference to the connector:
- https://central.ballerina.io/ballerinax/mysql/latest
- https://github.com/ballerina-platform/module-ballerinax-mysql

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -93,6 +93,7 @@ This list accepts and encourages pull requests. See [CONTRIBUTING](https://githu
 - [MariaDB Connector/J](https://github.com/mariadb-corporation/mariadb-connector-j) - LGPL-licensed MariaDB Client Library for Java Applications.
 - [mex-mariadb](https://github.com/markuman/mex-mariadb) - MIT licensed MariaDB/MySQL Client Library for GNU Octave and Matlab.
 - [MySQL C API](https://dev.mysql.com/downloads/c-api/) - Official C driver for MySQL.
+- [MySQL Connector/Ballerina](https://github.com/ballerina-platform/module-ballerinax-mysql) - Official Ballerina connector for MySQL.
```
Comment (dveeden) [inline README.md:96]:
```suggestion
- [ballerinax/mysql](https://github.com/ballerina-platform/module-ballerinax-mysql) - Official Ballerina connector for MySQL.
```

The "MySQL Connector/<lang>" naming is monstly/only used by official Oracle MySQL connectors. Let's try to use the name native to the language. For example I wouldn't cal "DBD::mysql" "MySQL Connector/Perl". However I don't think there's an official rule for this.

**Review by dveeden — APPROVED** [review]
This looks like a interesting new language.

The driver is Apache 2.0 licensed.

It looks like this uses MySQL Connector/J underneath right?

File: README.md (hunk 1 above)
Comment (SachinAkash01) [reply README.md]: Thanks for the suggestion! I have updated the file with reviewed changes and rearranged it according to the alphabetical order

**Comment by SachinAkash01** [comment]
> It looks like this uses MySQL Connector/J underneath right?

Yes you are correct!

**Review by shlomi-noach — APPROVED** [review]

**Pull request merged**